### ISSUE TITLE
Autodoc errors for documentation of technologies/thermal_network/network_layout

### ISSUE THREAD

**Issue by Jack-Hawthorne** [architecture-building-systems/CityEnergyAnalyst#1730]
The following error is raised by autodoc when creating documentation:

```
WARNING: autodoc: failed to import module u'main' from module u'cea.technologies.thermal_network.network_layout'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\sphinx\ext\autodoc\importer.py", line 154, in import_module
    __import__(modname)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\technologies\thermal_network\network_layout\main.py", line 5, in <module>
    from cea.technologies.thermal_network.network_layout.steiner_spanning_tree import calc_steiner_spanning_tree
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\technologies\thermal_network\network_layout\steiner_spanning_tree.py", line 6, in <module>
    from networkx.algorithms.approximation.steinertree import steiner_tree
RuntimeError: sys.path must be a list of directory names

WARNING: autodoc: failed to import module u'steiner_spanning_tree' from module u'cea.technologies.thermal_network.network_layout'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\sphinx\ext\autodoc\importer.py", line 154, in import_module
    __import__(modname)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\technologies\thermal_network\network_layout\steiner_spanning_tree.py", line 6, in <module>
    from networkx.algorithms.approximation.steinertree import steiner_tree
RuntimeError: sys.path must be a list of directory names
```
I'll look into it a bit deeper soon.
